pkg/scalers/aws/aws_sigv4.go

```diff
@@ -0,0 +1,88 @@
+package aws
+
```

Comment:
Please add here a comment with a few sentences that describe what is this file about